control-plane/rest/service/src/main.rs

```diff
@@ -33,6 +33,10 @@ pub(crate) struct CliArgs {
     #[structopt(long, short = "z", default_value = DEFAULT_GRPC_CLIENT_ADDR)]
     core_grpc: Uri,
 
+    /// The json gRPC Server URL or address to connect to the service.
+    #[structopt(long, short = "J", default_value = DEFAULT_JSON_GRPC_CLIENT_ADDR)]
+    json_grpc: Uri,
```

Comment:
This should be optional, we don't deploy json  grpc in prod by default

Reply:
Done. Made it an option.